internal/daysteps/daysteps.go

```diff
@@ -12,9 +17,63 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	input := strings.Split(data, ",")
+
+	if len(input) != 2 {
+		return 0, 0, fmt.Errorf("неверное количество данных: ожидается 2, получено %d", len(input))
+	}
+	if strings.TrimSpace(input[0]) == "" {
+		return 0, 0, fmt.Errorf("первое значение (количество шагов) пусто")
+	}
+	steps, err := strconv.Atoi(string(input[0]))
+	if err != nil {
+		return 0, 0, fmt.Errorf("не удалось преобразовать '%s' в число: %v", input, err)
+	}
+
+	if steps <= 0 {
+		return 0, 0, fmt.Errorf("ошибка колличество шагов не может равняться 0: %v", err)
+	}
+
+	duration, err := time.ParseDuration(input[1])
+	if err != nil {
+		return 0, 0, fmt.Errorf("ошибка преобразования времени: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("ошибка преобразования времени: %w", err)
```